MMR.Randomizer/Resources/REQ_CASUAL.txt

```diff
@@ -394,6 +382,10 @@
           "Powder Kegs as Explosives",
           "ItemPowderKeg",
           "MaskBlast"
+        ],
+        [
+          "MaskZora",
+          "GBT Iceless"
```

Comment:
This seems like a strange conditional for zora cape fairy fountain. How does this work?

Reply:
The trick GBT Iceless inherently includes a version of a Zora High Ledge Grab to get to the first red turnkey room, so I reluctantly added GBT Iceless as a conditional for the cape fairy because you can also get that with a Zora High Ledge Grab. Granted, it's probably not clear enough, so maybe it deserves its own trick, but I really don't like adding tricks for single checks.